src/engine/cache/file-cache.js

```diff
@@ -195,7 +201,12 @@ class FileCache extends BaseCache {
    * @returns {Boolean} - Cache empty or not
    */
   async isEmpty() {
-    const files = await fs.readdir(this.fileFolder)
+    let files = []
+    try {
+      files = await fs.readdir(this.fileFolder)
+    } catch (error) {
+      this.logger.error(error)
```

Comment:
a comment explaining why this can happen and why silently ignoring it is the right way to go would be more than helpful